array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,13 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  var onlyRight = [];
+
+  for (var i = array.length - count; i < array.length; i++) {
+    var value = array[i];
+
+    if (value || typeof value === 'boolean') {
```

Comment:
I feel like I'm in Groundhog Day. This conditional can be removed. Don't add code to functions unless you are certain it is necessary.